contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,888 @@
+        }
+
+        {
+            /// fUSDT
+            uint256 allowance = IERC20(USDT_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDT_ADDRESS
+            );
+
+            protocols[1] = F_USDT_ADDRESS;
+            tokens[1] = USDT_ADDRESS;
+            amounts[1] = 1_200_000 * 1e6; // 1.2M
+        }
+    }
+
+    // @notice Action 4: Reduce allowance of fUSDC and fUSDT
+    function action4() internal isActionSkippable(4) {
+        {
+            address cbBTC_USDC = getVaultAddress(29);
+
+            // Fee Handler Addresses
+            address FeeHandler = 0x7110cED08f0E26c14a2eF7A980c4F17C70aBa7c0;
+
+            // Remove handler as auth
+            VAULT_FACTORY.setVaultAuth(cbBTC_USDC, FeeHandler, false);
+        }
+
+        {
+            address cbBTC_USDT = getVaultAddress(30);
+
+            // Fee Handler Addresses
+            address FeeHandler = 0xf5fD6c6f936689018215CB10d7a5b99A43a39D28;
+
+            // Remove handler as auth
+            VAULT_FACTORY.setVaultAuth(cbBTC_USDT, FeeHandler, false);
+        }
+
+        {
+            address wBTC_USDT = getVaultAddress(22);
+
+            // Fee Handler Addresses
+            address FeeHandler = 0x6D90d460929b921f6C74838a9d25CC69B486D605;
+
+            // Remove handler as auth
+            VAULT_FACTORY.setVaultAuth(wBTC_USDT, FeeHandler, false);
+        }
+
+        {
+            address wBTC_USDC = getVaultAddress(21);
+
+            // Fee Handler Addresses
+            address FeeHandler = 0x5E5768B6b42c12dA8e75eb0AA6fD47Be33a5b24e;
+
+            // Remove handler as auth
+            VAULT_FACTORY.setVaultAuth(wBTC_USDC, FeeHandler, false);
+        }
+    }
+
+    // @notice Action 5: Remove Team Multisig as Auth on Unlaunched Vaults
+    function action5() internal isActionSkippable(5) {
+        {
+            address USDC_ETH__USDC_ETH = getVaultAddress(62);
+
+            VAULT_FACTORY.setVaultAuth(
+                USDC_ETH__USDC_ETH,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+        {
+            address WBTC_ETH__WBTC_ETH = getVaultAddress(63);
+
+            VAULT_FACTORY.setVaultAuth(
+                WBTC_ETH__WBTC_ETH,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+        {
+            address cbBTC_ETH__cbBTC_ETH = getVaultAddress(64);
+
+            VAULT_FACTORY.setVaultAuth(
+                cbBTC_ETH__cbBTC_ETH,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+        {
+            address USDe_USDC__USDe_USDC = getVaultAddress(65);
+
+            VAULT_FACTORY.setVaultAuth(
+                cbBTC_ETH__cbBTC_ETH,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+        {
+            address FLUID_ETH__ETH = getVaultAddress(75);
+
+            VAULT_FACTORY.setVaultAuth(FLUID_ETH__ETH, TEAM_MULTISIG, false);
+        }
```

Comment:
also need to Add vault 95: 0xbee8D906BAc00610D8056C88EeEc5E9e9D48104C
to remove team MS as auth